tests/manage/pv_services/space_reclaim/test_rbd_reclaimspace_cronjob.py

```diff
@@ -88,10 +88,11 @@ def test_rbd_space_reclaim_cronjob(self, pause_and_resume_cluster_load):
             )
             pod_obj.get_fio_results()
 
-        # Verify used size after IO
-        exp_used_size_after_io = used_size_before_io + (16 * self.pool_replica)
-        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
-        log.info(f"Used size after IO is {used_size_after_io}")
+            # Verify used size after IO
+            exp_used_size_after_io = used_size_before_io + (4 * self.pool_replica)
```

Comment:
(4 * self.pool_replica) 
why 4?

Reply:
> (4 * self.pool_replica) why 4?

4GB is the file size. Previously we were calculating total size after completing ios on all 4 files (4*4 = 16Gb), now I changed to calculate the size after completing ios immediately for all the files in each iteration, so it has to 4GB not 16GB